mappings/net/minecraft/client/util/Keyboard.mapping

```diff
@@ -2,6 +2,10 @@ CLASS cpj net/minecraft/client/util/Keyboard
 	FIELD a client Lcpk;
 	FIELD b repeatEvents Z
 	FIELD c clipboard Lcns;
+	FIELD d debugCrashStartTime J
+	FIELD e debugCrashLastLogTime J
+	FIELD f debugCrashCounter J
```

Comment:
What specifically does this count? It's not immediately clear from the code.

Reply:
It keep track of the seconds elapsed since the beginning of the countdown but is not really used for anything relevant.

Reply:
Yes, `debugCrashElapsedTime` is better. I changed it.